.licenserc.yaml

```
@@ -96,6 +96,16 @@ header: # `header` section is configurations for source codes license header.
     - '**/util/StringEncoding.java'
     - 'hugegraph-api/src/main/java/org/apache/hugegraph/opencypher/CypherOpProcessor.java'
     - 'hugegraph-api/src/main/java/org/apache/hugegraph/opencypher/CypherPlugin.java'
+    - 'hugegraph-pd/.mvn/wrapper/MavenWrapperDownloader.java'
```

```suggestion
    # TODO: temporarily added to the ignore list, need handle them before release
    - 'hugegraph-pd/.mvn/wrapper/MavenWrapperDownloader.java'
```

use `TODO` tag for reminder (so as other comments)